libs/cart/src/facades/cart/cart.facade.spec.ts

```diff
@@ -182,6 +182,336 @@ describe('DaffCartFacade', () => {
     });
   });
 
+  describe('featureLoading$', () => {
+    describe('when all the cart operations have completed', () => {
+      it('should return false', () => {
+        const expected = cold('a', {a: false});
+
+        expect(facade.featureLoading$).toBeObservable(expected);
+      });
+    });
+
+    describe('when the cart operations have not completed', () => {
+      beforeEach(() => {
+        facade.dispatch(new DaffCartLoad())
+      });
+
+      it('should return true', () => {
+        const expected = cold('a', {a: true});
+
+        expect(facade.featureLoading$).toBeObservable(expected);
+      });
+    });
+
+    describe('when the cart item operations have not completed', () => {
```

Comment:
Could you remove all of the subfield tests? It is already tested in the selectors, so it's all just extra here.

Reply:
It also tests the "are any" logic but I'll remove it if you really want me to.

Reply:
Yea, but that logic is actually done and tested in the selectors, not the facade. Really, we just want to verify that the facade is giving the value of the selector we're setting it to. We don't need to retest all of the selector logic.

Reply:
ah my bad you were specifically talking about the facade tests. I should have understood. removed the extra facade test cases.